apps/src/templates/census2017/YourSchool.jsx

```diff
@@ -124,11 +124,7 @@ class YourSchool extends Component {
           )}
         <h1 style={styles.heading}>{i18n.yourSchoolHeading()}</h1>
         <h3 style={styles.description}>{i18n.yourSchoolDescription()}</h3>
-        <ProfessionalLearningApplyBanner
-          nominated={false}
-          useSignUpText={true}
-          style={styles.banner}
-        />
+        <AtHomeBanner />
```

Comment:
How does this hold up on mobile?

Reply:
@Erin007 I added a mobile simulator screenshot to the description!